### PR TITLE
the TISCORD PR

### PR DESCRIPTION
**Pull request by hkubus**
this is very much wip
removes messagecreate and every sign of prefix commands
removes antibot because it doesnt work AT ALL
removes ping (ping is added in tiscord 2.0.0

### CODE REVIEW

File: src/classes/Client.ts [hunk 1]
```diff
@@ -31,7 +30,6 @@ export default class FluorineClient extends Client {
     cooldowns = new CooldownHandler(this);
 
     economy = new EconomyModule(this);
-    phishing = new PhishingModule(this);
```
Comment (splatterxl) [inline src/classes/Client.ts:34]: Phishing is Gone

File: src/classes/modules/CasesModule.ts [hunk 2]
```diff
@@ -80,13 +81,13 @@ export default class CasesModule {
...
-            const channel = guildObj.channels.cache.get(logsChannel.toString());
+            const channel = (await guildObj.channels.get(logsChannel.toString())) as Channel;
 
-            if (!channel?.isTextBased()) {
+            if (channel?.type !== ChannelType.GuildText) {
                 return;
             }
 
-            channel.send({ embeds: [embed] });
+            (channel as TextChannel).send({ embeds: [embed.toJSON()] });
```
Comment (splatterxl) [inline src/classes/modules/CasesModule.ts:90]: this shouldn't need to be casted. fix your types

File: src/commands/ban.ts [hunk 3]
```diff
@@ -29,7 +31,7 @@ export async function run(client: FluorineClient, interaction: ChatInputCommandI
         });
     }
 
-    if (!member.bannable) {
+    if (!member.permissions.has('ADMINISTRATOR')) {
```
Comment (splatterxl) [inline src/commands/ban.ts:34]: what

File: src/commands/deploy/create.ts [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
-    await interaction.deferReply();
+    await interaction.defer();
 
     const name = interaction.options.getString('command');
     let guildId = interaction.options.getString('guild');
     const command = client.commands.chatInput.get(name) ?? client.commands.contextMenu.get(name);
 
     if (!command && name !== 'all') {
-        return interaction.editReply(`Command \`${name}\` not found.`);
+        console.log('Deez');
```
Comment (splatterxl) [inline src/commands/deploy/create.ts:15]: Deez

File: src/classes/modules/CasesModule.ts (hunk 2 above)
Comment (hkubus) [reply src/classes/modules/CasesModule.ts:90]: hi theyre fixed in tiscord dev loooooooooool

File: src/classes/Client.ts (hunk 1 above)
Comment (hkubus) [reply src/classes/Client.ts:34]: hi Yes

File: src/commands/ban.ts (hunk 3 above)
Comment (hkubus) [reply src/commands/ban.ts:34]: hi tiscord doesnt have bannable

Comment (xiboon) [reply src/commands/ban.ts:34]: yet

File: src/commands/deploy/create.ts (hunk 4 above)
Comment (hkubus) [reply src/commands/deploy/create.ts:15]: nuts

**Comment by SuperchupuDev** [comment]
typescript

**Comment by splatterxl** [comment]
typescript is love typescript is life

**Pull request closed without merging**